Reply/app/src/main/res/values/dimens.xml

```diff
@@ -33,6 +33,7 @@
     <dimen name="email_recipient_card_popup_elevation">6dp</dimen>
     <dimen name="email_recipient_card_popup_elevation_compat">3dp</dimen>
     <dimen name="email_star_image_size">40dp</dimen>
+    <dimen name="email_card_view_bottom_padding">120dp</dimen>
```

Comment:
Just a note for the future. We will need to calculate the bottom padding programmatically since the height of the BottomNavigationView (and BottomAppBar currently) will change depending on the bottom window inset ([this blog post helps describe what insets are](https://medium.com/androiddevelopers/windowinsets-listeners-to-layouts-8f9ccc8fa4d1)).

Let wait to do that once we change to using a BottomNavigationView though